definitions/pipelines/rnaseq.cwl

```diff
@@ -129,10 +132,16 @@ steps:
             bam: merge/merged_bam
         out:
             [indexed_bam]
+    mark_dup:
```

Comment:
We don't need to output a separate markdup bam - just need to make sure the final bam is derived from it (as you did above!)

Reply:
my bad - I misread this. it's fine!